pymongo/database.py

```diff
@@ -942,20 +1015,25 @@ def cursor_command(
 
     def _retryable_read_command(
         self,
-        command,
-        value=1,
-        check=True,
-        allowable_errors=None,
-        read_preference=None,
-        codec_options=DEFAULT_CODEC_OPTIONS,
-        session=None,
-        **kwargs,
-    ):
+        command: Union[str, MutableMapping[str, Any]],
+        value: int = 1,
+        check: bool = True,
+        allowable_errors: Optional[Sequence[Union[str, int]]] = None,
+        read_preference: Optional[_ServerMode] = None,
+        codec_options: CodecOptions = DEFAULT_CODEC_OPTIONS,
+        session: Optional[ClientSession] = None,
+        **kwargs: Any,
```

Comment:
I think all of these arguments can be removed except `command` and `session`. _retryable_read_command is only used once: `admin._retryable_read_command(cmd, session=session)`

Reply:
makes sense. removed